package.json

```diff
@@ -2,12 +2,11 @@
   "name": "vott-react-typescript",
   "version": "0.1.0",
   "private": true,
-  "main": "build/electron/bundle.js",
+  "main": "src/electron/main.ts",
```

Comment:
This causes the app startup to fail for devs running `npm start`.  We need an electron bundle to be used as the start script.